core/src/apps/management/reset_device/__init__.py

```diff
@@ -228,3 +252,16 @@ async def backup_seed(
         await _backup_slip39_advanced(ctx, mnemonic_secret)
     else:
         await layout.bip39_show_and_confirm_mnemonic(ctx, mnemonic_secret.decode())
+
+
+async def sd_card_backup_seed(
+    ctx: Context, mnemonic_secret: bytes
+) -> None:
+    from storage.sd_seed_backup import set_sd_seed_backup
+    from apps.common.sdcard import ensure_sdcard
+
+    # Ensure sd card
+    await ensure_sdcard(ctx)
```

Comment:
create a separate `ensure_sdbackup` or something that will (for now) only throw an error if sd card is inaccessible, but it will format without prompt, etc.